core/api/pom.xml

```diff
@@ -23,6 +23,11 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.alibaba</groupId>
+            <artifactId>transmittable-thread-local</artifactId>
+            <version>2.2.0</version>
```

Comment:
Please manage the version in dependencyManagement section in parent pom.

Reply:
done